src/oatpp-swagger/oas3/Generator.cpp

```diff
@@ -102,6 +102,9 @@ oatpp::Object<Schema> Generator::generateSchemaForSimpleType(const Type* type, T
     if(!property->info.description.empty()) {
       result->description = property->info.description.c_str();
     }
+    if (defaultValue != nullptr) {
+      result->defaultValue = *defaultValue;
```

Comment:
Just `result->defaultValue = defaultValue;` (without `*`) as we are using oatpp types now.